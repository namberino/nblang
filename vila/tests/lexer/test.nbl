// comment
{()} // grouping

!*+-/=<> <= == += -= *= /= // operators

(1 + 2) * 3 / 4

<>{},.-+;/*

> >= < <= = == != ! 

txt "txt" 1

and class else false fun for if nil or print return super this true mut while
