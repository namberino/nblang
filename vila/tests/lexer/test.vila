// comment
{()} // grouping

!*+-/=<> <= == += -= *= /= // operators

(1 + 2) * 3 / 4

<>{},.-+;/*

> >= < <= = == != ! 

txt "txt" 1

và lớp thì sai hàm với nếu trống hoặc in trả gốc đây đúng biến khi
