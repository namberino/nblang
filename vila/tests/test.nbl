print("Hello world!");
// print(clock());
// print(time());
