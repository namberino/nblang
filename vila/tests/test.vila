in("Hello world!");
// in(đồng_hồ());
// in(thời_gian());
