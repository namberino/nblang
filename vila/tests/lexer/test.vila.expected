On line: 2, Error at ')': Expected an expression
On line: 8, Error at '/': Expected an expression
On line: 14, Error at 'else': Expected class name
On line: 14, Error at 'for': Expected function name
On line: 14, Error at 'nil': Expected '(' after 'if' statement
On line: 14, Error at 'return': Expected '(' after 'print' statement
On line: 14, Error at 'while': Expected variable name
On line: 15, Error at end: Expected '}' after block
